app/javascript/credential.js

```diff
@@ -1,13 +1,22 @@
 import * as Encoder from "encoder";
 
+function getCSFRToken() {
```

Comment:
Typo: Should be CSRF.

Catching it because also happens to me from to time :smile:

Reply:
🤦‍♂️